src/Controls/src/Core.Design/FlexEnumDesignTypeConverters.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Maui.Controls.Design
+{
+	public class FlexJustifyTypeConverter : KnownValuesDesignTypeConverter
+	{
+		protected override bool ExclusiveToKnownValues => true;
+
+		protected override string[] KnownValues
+			=> new[] {
+				"Start",
```

Comment:
So all the flex-* values that the [FlexEnumConverters](https://github.com/dotnet/maui/blob/main/src/Core/src/Converters/FlexEnumsConverters.cs) allow aren't needed here because only CSS can use them but not XAML? How do you actually prevent those values from being used in XAML?

Reply:
we don't prevent it's usage at this time unfortunately, but we frown at it. What I'm realising right now is that we prevent XAML compilation of that value, due to [TypeConverter] attached.

Reply:
I know, and I didn't understand how XAML was going to not support those values there either, but now I have my answer. With the new design time TypeConverters the "frown at it" will become much more visible to everyone. I guess we'll see if our users will frown at that. :-)